includes/admin/settings/class-settings-recurring.php

```diff
@@ -15,34 +15,16 @@ class Give_Settings_Recurring_Donations_Core extends Give_Settings_Page
     public function __construct()
     {
         $this->id = 'recurring';
-        $this->label = __('Recurring Donations', 'give');
+        $this->label = sprintf(
+            __('%s Recurring Donations %s', 'give'),
+            '<svg style="display: inline-block; vertical-align: middle; margin: 0 5px 1px 0; " width="12" height="12" viewBox="0 0 12 12" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M7.16675 0.75002C7.16675 0.427854 7.42792 0.166687 7.75008 0.166687H11.2501C11.5722 0.166687 11.8334 0.427854 11.8334 0.75002L11.8334 4.25002C11.8334 4.57219 11.5722 4.83335 11.2501 4.83335C10.9279 4.83335 10.6667 4.57219 10.6667 4.25002L10.6667 2.15831L6.99589 5.82917C6.76809 6.05697 6.39874 6.05697 6.17094 5.82917C5.94313 5.60136 5.94313 5.23201 6.17094 5.00421L9.84179 1.33335H7.75008C7.42792 1.33335 7.16675 1.07219 7.16675 0.75002Z" fill="#0E0E0E"/><path d="M3.52599 1.33335L4.83342 1.33335C5.15558 1.33335 5.41675 1.59452 5.41675 1.91669C5.41675 2.23885 5.15558 2.50002 4.83342 2.50002H3.55008C3.05041 2.50002 2.71074 2.50047 2.44819 2.52193C2.19245 2.54282 2.06166 2.58069 1.97043 2.62718C1.7509 2.73903 1.57243 2.91751 1.46057 3.13703C1.41409 3.22827 1.37622 3.35905 1.35532 3.61479C1.33387 3.87735 1.33342 4.21702 1.33342 4.71669V8.45002C1.33342 8.94969 1.33387 9.28936 1.35532 9.55192C1.37622 9.80766 1.41409 9.93844 1.46057 10.0297C1.57243 10.2492 1.7509 10.4277 1.97043 10.5395C2.06166 10.586 2.19245 10.6239 2.44819 10.6448C2.71074 10.6662 3.05041 10.6667 3.55008 10.6667H7.28342C7.78309 10.6667 8.12276 10.6662 8.38531 10.6448C8.64105 10.6239 8.77184 10.586 8.86307 10.5395C9.08259 10.4277 9.26107 10.2492 9.37292 10.0297C9.41941 9.93844 9.45728 9.80766 9.47818 9.55192C9.49963 9.28936 9.50008 8.94969 9.50008 8.45002V7.16669C9.50008 6.84452 9.76125 6.58335 10.0834 6.58335C10.4056 6.58335 10.6667 6.84452 10.6667 7.16669V8.47413C10.6668 8.94369 10.6668 9.33123 10.641 9.64692C10.6142 9.97481 10.5567 10.2762 10.4124 10.5593C10.1887 10.9984 9.83177 11.3553 9.39273 11.579C9.10961 11.7233 8.8082 11.7808 8.48031 11.8076C8.16462 11.8334 7.77708 11.8334 7.30752 11.8334H3.52598C3.05642 11.8334 2.66888 11.8334 2.35318 11.8076C2.02529 11.7808 1.72388 11.7233 1.44077 11.579C1.00173 11.3553 0.644771 10.9984 0.421067 10.5593C0.276814 10.2762 0.219318 9.97481 0.192528 9.64692C0.166735 9.33122 0.166741 8.94368 0.166748 8.47412V4.6926C0.166741 4.22303 0.166735 3.83548 0.192528 3.51979C0.219318 3.1919 0.276814 2.89049 0.421067 2.60738C0.644771 2.16833 1.00173 1.81138 1.44077 1.58767C1.72388 1.44342 2.02529 1.38592 2.35318 1.35913C2.66888 1.33334 3.05642 1.33335 3.52599 1.33335Z" fill="#50575e"/></svg>',
+            '<span style="display: inline-block; vertical-align: middle; margin: 0 0 2px 8px; padding: 4px 8px; background:#F2CC0C; font-size: 10px; border-radius: 4px;"><strong>RECOMMENDED</strong></span>'
```

Comment:
"RECOMMENDED" needs to be translatable.

Reply:
Resolved: https://github.com/impress-org/givewp/pull/6794/commits/39bd4cb412e2b3fc07a7e8f5167ca9afbfe1532d